packages/visx-xychart/src/components/series/private/Bars.tsx

```diff
@@ -7,21 +9,43 @@ export default function Bars({
   horizontal,
   xScale,
   yScale,
-  ...rectProps
-}: BarsProps<any, any>) {
-  const isFocusable = Boolean(rectProps.onFocus || rectProps.onBlur);
+  radius,
+  radiusAll,
+  radiusTop,
+  radiusRight,
+  radiusBottom,
+  radiusLeft,
+  ...restProps
+}: BarsProps<AxisScale, AxisScale>) {
+  const isFocusable = Boolean(restProps.onFocus || restProps.onBlur);
   return (
     // eslint-disable-next-line react/jsx-no-useless-fragment
     <>
-      {bars.map(({ key, ...barProps }) => (
-        <rect
-          key={key}
-          className="visx-bar"
-          tabIndex={isFocusable ? 0 : undefined}
-          {...barProps}
-          {...rectProps}
-        />
-      ))}
+      {bars.map(({ key, ...barProps }) =>
+        radius == null ? (
+          <rect
+            key={key}
+            className="visx-bar"
+            tabIndex={isFocusable ? 0 : undefined}
+            {...barProps}
+            {...restProps}
+          />
+        ) : (
+          <BarRounded
+            key={key}
+            className="visx-bar"
+            tabIndex={isFocusable ? 0 : undefined}
+            radius={radius}
```

Comment:
I forgot when `BarRounded` was added why `radiusTop` was called `top` instead of `radiusTop`.

Reply:
it was contributed by someone else. I think in the context of `BarRounded`, `top/right/bottom/left/topRight/etc` are more self evident (esp with types), but not as obvious here.